browser/brave_ads/ads_service_factory.cc

```diff
@@ -98,7 +100,8 @@ KeyedService* AdsServiceFactory::BuildServiceInstanceFor(
           std::make_unique<AdsTooltipsDelegateImpl>(profile),
 #endif
           std::make_unique<DeviceIdImpl>(), history_service, rewards_service,
-          notification_ad_async_data_store);
+          notification_ad_async_data_store, g_browser_process->local_state(),
```

Comment:
You can access prefs from `AdsServiceImpl` using `profile_->GetPrefs();`

Reply:
Wouldn't that return the profile prefs? I'm trying to use the local prefs for this use case.

Reply:
why would you use local prefs? AdsService is per-profile

Reply:
So we can record single totals of campaign views/clicks/landings from all profiles. If we stored the totals in profile prefs, the metric values may be overwritten with the profile-specific totals whenever the user switches profile.